server/pulp/server/async/worker_watcher.py

```diff
@@ -78,9 +78,6 @@ def handle_worker_heartbeat(event):
         msg = _("New worker '%(worker_name)s' discovered") % event_info
```

Comment:
Once PR #2907 goes through and you rebase onto it, you will want to replace the magic number "30" with the constant ```PULP_PROCESS_HEARTBEAT_INTERVAL``` from ```pulp.common.constants```.  FYI, that interval is 5 seconds, which is the new timing for it.  In case you need to change comments/docs.